src/app/reskin/components/stepper/deposit-timeline.tsx

```diff
@@ -0,0 +1,70 @@
+"use client";
+import { StepContent, StepTitle } from "./timeline";
+import { defineStepper } from "@stepperize/react";
+
+import { AmountDescription } from "./deposit/amount";
+import { StatusDescription } from "./deposit/status";
+import { useMediaQuery } from "@/hooks/use-media-query";
+
+function AddressDescription({}) {
+  return (
+    <span className="opacity-60">
+      sBTC will be sent to a STX address. Connecting a wallet will auto-fill
+      this in, but feel free to submit another address.
+    </span>
+  );
+}
+
+const { useStepper, utils } = defineStepper(
+  {
+    id: "amount",
+    title: "Select Withdrawal Amount",
+    description: AmountDescription,
+  },
+  {
+    id: "address",
+    title: "Provide a Deposit Address",
+    description: AddressDescription,
+  },
+  {
+    id: "status",
+    title: "Operation Status:",
+    description: StatusDescription,
+  },
+);
+
+function DepositTimeline() {
+  const stepper = useStepper();
+
+  const isDesktop = useMediaQuery("(min-width: 1024px)");
+  const currentIndex = utils.getIndex(stepper.current.id);
+
+  return (
+    isDesktop && (
+      <div className="pt-6 pb-10 px-8 border rounded-2xl border-black border-opacity-20 dark:border-white dark:border-opacity-20 w-96">
```

Comment:
nit: do we need `dark:border-opacity-20` given we already have `border-opacity-20`?